packages/flutter/test/rendering/editable_gesture_test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class FakeEditableTextState extends TextSelectionDelegate {
```

Comment:
nit: move this to the end of the file (or you might be able to remove it entirely by using a Mockito mock).

Reply:
In fact it's the same one as the one under `editable_test.dart` so I can just used that one instead of duplicate it.

Reply:
Ok look like it's not allowed ^^ changing to mockito mock then :)